content/contribute/project-information/working-groups.md

```diff
@@ -7,33 +7,39 @@ weight = 1
 
 Working groups are temporary community initiatives devoted to particularly complex, large, or important tasks. This might include:
 
-- undertaking a major refactor
-- adding an unusually complex new feature
-- upstreaming an ecosystem crate
-- preparing for a new release
-- bootstrapping new major documentation efforts
+- Undertaking a major refactor.
```

Comment:
I don't like this style of list for short bullet points like this. These lists feel more like an extension of / reformatting of a simple comma separated list in a sentence. Can you explain why you prefer it?

Reply:
Because each bullet point is effectively the start of a new sentence in my opinion, thus should follow the appropriate grammar rules. I can change it to act as a formatted comma delimited list if you want though.